### ISSUE TITLE
Mock one specific Meteor method

### ISSUE THREAD

**Issue by Sanjo** [meteor-velocity/velocity-helpers#3]
```js
spyOnMethod('myMeteorMethod').and.returnValue('result');
```

should be available and not affect other Meteor methods.

**Comment by Sanjo** [comment]
Here is how I do this without sugar:

```js
var call = Meteor.call;
// data is the argument of the method we mock
spyOn(Meteor, 'call').and.callFake(function (method, data, callback) {
  if (method === 'myMeteorMethod') {
    callback(null, 'result');
  } else {
    call.apply(Meteor, arguments);
  }
});
```

**Comment by Sanjo** [comment]
Released with 0.1.0
